src/collection.ts

```diff
@@ -0,0 +1,320 @@
+import axios from "axios";
```

Comment:
you may need to install all dependencies gh action workflow, maybe not just the developer ones